client/deposits/strings.ts

```diff
@@ -32,3 +32,106 @@ export const depositStatusLabels: Record<
 	canceled: __( 'Canceled', 'woocommerce-payments' ),
 	failed: __( 'Failed', 'woocommerce-payments' ),
 };
+
+/**
+ * Mapping of payout failure code to display string.
+ */
+export const payoutFailureMessages: Record< PayoutFailureCode, string > = {
```

Comment:
Looking at https://docs.stripe.com/api/payouts/object#payout_object-failure_code, these are missing from this mapping:
- incorrect_account_holder_address
- incorrect_account_holder_tax_id
- invalid_account_number_length

Is that intentional?

Reply:
I have added these three failure codes to the map here - [072cb46](https://github.com/Automattic/woocommerce-payments/pull/10462/commits/072cb4605b39c70b95de158761c8fd42dbd759e0) .. Thanks for spotting it!